src/pages/settings/Wallet/WalletEmptyState.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import IllustratedHeaderPageLayout from '../../../components/IllustratedHeaderPageLayout';
+import themeColors from '../../../styles/themes/default';
+import useLocalize from '../../../hooks/useLocalize';
+import * as LottieAnimations from '../../../components/LottieAnimations';
+import Button from '../../../components/Button';
+import Navigation from '../../../libs/Navigation/Navigation';
+import SCREENS from '../../../SCREENS';
+import ROUTES from '../../../ROUTES';
+import Icon from '../../../components/Icon';
+import styles from '../../../styles/styles';
+import * as Illustrations from '../../../components/Icon/Illustrations';
+import variables from '../../../styles/variables';
+import Text from '../../../components/Text';
+
+const propTypes = {
+    /** What to do when a menu item is pressed */
+    onAddPaymentMethod: PropTypes.func.isRequired,
+};
+
+function WalletEmptyState(props) {
```

Comment:
Can't this be deconstructed here?

`function WalletEmptyState({ onAddPaymentMethod })`